### PR TITLE
Update i18n version in sign in widget to 30.2.0 

### PR DESCRIPTION
**Pull request by nikhilvenkatraman-okta**
Update i18n version in okta-sign in-widget to 30.2.0 . This is pull the changes from https://github.com/okta/npm/pull/760 into sign in widget.

@mauriciocastillosilva-okta @samerfanek-okta @hor-kanchan-okta

### CODE REVIEW

File: node_modules/@okta/i18n/package.json [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
+    "raw": "@okta/i18n@30.2.0",
+    "rawSpec": "30.2.0",
     "scope": "@okta",
-    "spec": "28.25.0",
+    "spec": "30.2.0",
     "type": "version"
   },
   "_requiredBy": [
-    "/"
+    "#USER"
```
Comment (hor-kanchan-okta) [inline node_modules/@okta/i18n/package.json]: What does this mean?

Comment (nikhilvenkatraman-okta) [inline node_modules/@okta/i18n/package.json]: @hor-kanchan-okta not sure I need to check. This file was auto updated using a command.

Comment (mauriciocastillosilva-okta) [inline node_modules/@okta/i18n/package.json]: This should not change, it should remain "/", this is most likely the reason Travis failed.

File: node_modules/@okta/i18n/package.json [hunk 2]
```diff
@@ -1,30 +1,31 @@
...
+      "@okta/i18n@30.2.0",
       "/home/vagrant/okta/okta-signin-widget"
     ]
   ],
-  "_from": "@okta/i18n@28.25.0",
-  "_id": "@okta/i18n@28.25.0",
+  "_from": "@okta/i18n@30.2.0",
+  "_id": "@okta/i18n@30.2.0",
   "_inCache": true,
+  "_installable": true,
```
Comment (mauriciocastillosilva-okta) [inline node_modules/@okta/i18n/package.json]: this probably should not be here.

**Comment by nikhilvenkatraman-okta** [comment]
The 3 tests that are failing are related to errorCode `E0000068`. Now that we have an entry in the properties file for `errors.E0000068` its picking up the errorSummary at the root level and not inside errorCauses. This was kind of expected. For now i can fix the test. I think that these errorSummaries are always in English today. @hor-kanchan-okta  @mauriciocastillosilva-okta Let me know what you guys think.

**Comment by nikhilvenkatraman-okta** [comment]
I have removed the entry for error code E0000068 from login.properties. Will create a new PR to update the sign in widget i18n version.

**Pull request closed without merging**